messages/sensor/lidar.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "LIDAR",
+    "description": "Data from a LIDAR sensor",
+    "default": {},
+    "required": [
+        "pose",
+        "points"
```

Comment:
I think this is good for the required properties, but perhaps we should add optional properties for normal vectors and color as well?